### DISCUSSION TITLE
Read from smartcontract on front-end. Lesson 13

### DISCUSSION THREAD

**Discussion by NarithChesda** [smartcontractkit/full-blockchain-solidity-course-py#1361] [Q&A] [answered]
I'm struggling on front-end part. I've never done any web development before. I made a hook to read a stakingBalance for Lesson 13, 
is there anybody that have already done it? Can you help share your code so I can learn from the example?

**Comment by cromewar** [comment, marked as answer]
Does this help you?
https://github.com/retro-cats/retro-cats-front-end-js

**Reply by NarithChesda** [reply]
I figured how to read from smart contract using usedapp, but thank you! it's good to see other people's work that is taking different approach.
